proposals/repo-to-tar/README.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Package Repository to Tar"
+authors: [ "Ashish Kumar <ashishndiitr@gmail.com>" ]
+status: "draft"
+approvers: [ ]
+---
+
+# <Package Repository to Tar>
+
+## Problem Statement
+
+Presently imgpkg does not support the creation of tar file directly.
+In order to obtain a tar file, one needs to first push the bundle image to the registry and then leverage the [command](https://carvel.dev/imgpkg/docs/v0.37.x/air-gapped-workflow/#option-2-with-intermediate-tarball) for copying as tar in the air gapped workflow.
+
+For example, In order to obtain a tar of bundle image of examples/basic-step-2, we need run the following commands in succession :<br>
+
+`imgpkg push -b index.docker.io/user1/simple-app-bundle:v1.0.0 -f examples/basic-step-2` <br>
+`imgpkg copy -b index.docker.io/user1/simple-app-bundle:v1.0.0 --to-tar /tmp/my-image.tar`
```

Comment:
This is not the same as the tar we are trying to create here.
This tar will have all dependent images zipped up as well. However, our tar will just have the layer with the configuration. (You are on the right track with this as per our previous discussion)